src/css/utility.css

```diff
@@ -1082,6 +1082,20 @@
   }
 }
 
+@media screen and (min-width: 64rem) {
+  .d-none-laptop-onwards {
+    display: none !important;
+  }
+
+  .d-block-laptop-onwards {
+    display: block !important;
+  }
+
+  .d-flex-laptop-onwards {
+    display: flex !important;
```

Comment:
This !important is really needed here ? I think you should look from the specificity perspective first.

Reply:
this is a utility class. intended to overwrite any existing display property regardless